tests/test_client.py

```diff
@@ -1,28 +1,30 @@
-import re
-
 import pytest
+from dacite import from_dict
 
 from quetz_client.client import Channel, ChannelMember, QuetzClient
 
 from .conftest import temporary_package_file
 
 
-def test_yield_channels(quetz_client):
-    expected_channel_names = ("a", "b", "c")
-    channels = list(quetz_client.yield_channels(limit=2))
+def test_yield_channels(client: QuetzClient, three_channels):
+    channels = list(client.yield_channels(limit=2, query="c-"))
     assert len(channels) == 3
     assert isinstance(channels[0], Channel)
-    assert {channel.name for channel in channels} == set(expected_channel_names)
+    assert {from_dict(Channel, c) for c in three_channels} == set(channels)
 
 
-def test_yield_channel_members(quetz_client: QuetzClient, expected_channel_members):
+def test_yield_channel_members(
+    client: QuetzClient, expected_channel_a_members, live_post_channel_a_members
+):
     channel = "a"
-    channel_members = set(quetz_client.yield_channel_members(channel=channel))
-    assert {ChannelMember(**ecm) for ecm in expected_channel_members} == channel_members
+    channel_members = set(client.yield_channel_members(channel=channel))
+    assert {
+        from_dict(ChannelMember, ecm) for ecm in expected_channel_a_members
+    } == channel_members
 
 
-def test_yield_users(quetz_client: QuetzClient, expected_users):
-    users = list(quetz_client.yield_users())
+def test_yield_users(client: QuetzClient, expected_users):
```

Comment:
are these live or mock tests?

Reply:
The tests which don't have `live` or `mock` in their names run once for each.